CMakeLists.txt

```diff
@@ -39,7 +39,13 @@ endif()
 if (MSVC)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /EHsc")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /fp:precise") # floating point model: precise
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /GT") # fiber safe optimizations
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /arch:AVX2")
```

Comment:
The dyanmic recompiler seems to use instruction set extensions based on their availability.
Unless see a measurable performance improvement by compiling the C++ code with AVX2, this probably should not be made a default flag.

Reply:
fixed